src/routes/measurement-threshold/find.ts

```diff
@@ -0,0 +1,110 @@
+import Joi from 'joi';
+import { Edge } from '../../models/edge';
+import { Pin } from '../../models/pin';
+import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+
+import { PinGroup } from '../../models/pin-group';
+
+import { TableQuery, EffectiveTableQuery, tableQuerySchemaGenerator } from '../../comms/table-controller';
+import { Quantity } from '../../models/quantity';
+import { stringBeforeV7ElseStringOrTranslationSchema } from '../../models/string-or-translations';
+import { schema as siNumberSchema, SiNumber } from '../../models/si-number';
+
+interface Query extends TableQuery {
+  quantityHashId?: string | null;
+  pinGroupHashId?: string | null;
+  edgeHashId?: string | null;
+  pinHashId?: string | null;
+  type?: 'quantity' | 'port';
```

Comment:
I meant putting them in the search string (so not listed explicitly here). That offers more flexibility (in line with the multiselect autocomplete of the front-end), for example to provide `quantity:aaa quantity:bbb` to get an OR for both.